catboost/libs/column_description/cd_parser.cpp

```diff
@@ -70,6 +70,9 @@ namespace {
                 if (type == "Target") {
                     type = "Label";
                 }
+                if (type == "DocId") {
```

Comment:
The same logic should be added where output columns are parsed in eval_result.cpp

Reply:
Hm, it is tricky. I was expected that `column_descriptor/` is the only place where string names of column types are used. I think it would be better to pass `TVector<TColumn>` instead of `TVector<TString>` to `OutputEvalResultToFile()`.

Reply:
> I think it would be better to pass TVector<TColumn> instead of TVector<TString> to OutputEvalResultToFile().

I agree but this is more involved refactoring outside of this particular change.